src/github/services/dead-letter-queue.services.spec.ts

```diff
@@ -0,0 +1,82 @@
+import {fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {ConfigService} from '../../app/config.service';
+import {DeadLetterQueueService} from './dead-letter-queue.service';
+import {of} from 'rxjs/observable/of';
+import {HttpClient, HttpClientModule} from '@angular/common/http';
+import {ACTION_PRIORITY_IGNORE, ACTION_PRIORITY_NOW} from '../../domain/action-item';
+import {Observable} from 'rxjs/Observable';
+
+describe('dead letter queue service', () => {
+
+  const TEST_TEAM = 'test-team-name';
+  const TEAM_SVC = 'super-svc';
+  const TEAM_SCS = 'best-scs';
+  const SVC_ZONE = 'murika';
+
+  let service: DeadLetterQueueService;
+  let http: HttpClient;
+
+  beforeEach(() => {
+   const fakeConfigSvc = {
+      vsts: {
+        team: TEST_TEAM
+      }
+    };
+
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientModule
+      ],
+      providers: [
+        { provide: ConfigService, useValue: fakeConfigSvc },
+        DeadLetterQueueService
+      ]
+    });
+
+    service = TestBed.get(DeadLetterQueueService);
+    spyOn(service, 'getAllConfigurations').and.returnValue({
+      'test-team-name': [{
```

Comment:
@Blackbaud-ColbyWhite is there a way for me to use my constant here or no?  kind sucks to maintain the string in two places...

Reply:
`spyOn(service, 'getAllConfigurations').and.returnValue({[TEST_TEAM]: blah})` should work.

Reply:
let me try the `[...]` bit and see if things still go smoothyl

Reply:
totally worked